.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Please update this template with something that matches your PR -->
```

Comment:
We should try to use the [GitHub recognized keywords](https://help.github.com/en/articles/closing-issues-using-keywords).

Reply:
it's already done with the key word `close` ;)